registrations/signals.py

```diff
@@ -17,6 +29,22 @@ def _signup_or_group_post_delete(instance: Union[SignUp, SignUpGroup]) -> None:
         contact_person.send_notification(SignUpNotificationType.CANCELLATION)
 
 
+def _send_event_cancellation_notification(event):
+    registration_ids = Registration.objects.filter(event_id=event.pk).values_list(
+        "pk", flat=True
+    )
+
+    for contact_person in SignUpContactPerson.objects.filter(
+        Q(email__isnull=False)
+        & ~Q(email__exact="")
```

Comment:
```suggestion
        & ~Q(email="")
```
No need for `__exact` because it is used when no lookup type is specified.